crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -568,16 +568,19 @@ where
                     // de-synchronization between on-chain and off-chain databases.
                     if let Err(e) = result {
                         tracing::error!("Error processing block: {:?}", e);
-                        should_continue = self.continue_on_error;
+                        if self.continue_on_error {
+                            TaskNextAction::Continue
```

Comment:
why not `ErrorContinue` here?

Reply:
Yeah. Good idea. I was going to say that I want to maintain the old behavior, but that's essentially the same and cleaner.